MonkeyLoader.GamePacks.ResoniteModLoader/Util.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MonkeyLoader.Logging;
+using System.Collections.Generic;
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Linq;
+
+namespace ResoniteModLoader;
+
+internal static class Util
+{
+    /// <summary>
+    /// Get the Logger for the mod from a stack trace.
+    /// </summary>
+    /// <param name="stackTrace">A stack trace captured by the callee</param>
+    /// <returns>The executing mod's Logger, or ModLoader.Logger if none found</returns>
+    internal static Logger GetLoggerFromStackTrace(StackTrace stackTrace)
```

Comment:
you can just move this into the ResoniteMod or RmlMod class instead, no need for an extra util class. If you move it into the RmlMod class, the assembly map can be private too, as it's not accessed from anywhere else.